components/builder-depot/src/error.rs

```diff
@@ -25,26 +25,35 @@ use hab_net;
 use hyper;
 use iron;
 use protobuf;
+use rusoto_s3;
 use zmq;
 
 #[derive(Debug)]
 pub enum Error {
     BadPort(String),
     ChannelAlreadyExists(String),
     ChannelDoesNotExist(String),
+    CreateBucketError(rusoto_s3::CreateBucketError),
     DepotClientError(depot_client::Error),
     HabitatCore(hab_core::Error),
     HabitatNet(hab_net::error::LibError),
-    NetError(hab_net::NetError),
```

Comment:
Not sure where that went... I don't recall removing this.

Reply:
Perhaps I removed this because it wasn't being used. Im not seeing any errors or warnings in compilation about it so I'm gonna go with it.